lib/hocon/impl/simple_config_list.rb

```diff
@@ -149,11 +216,122 @@ def render_value_to_sb(sb, indent_size, at_root, options)
     end
   end
 
-  def new_copy(origin)
-    Hocon::Impl::SimpleConfigList.new(origin, @value)
+  def contains?(o)
```

Comment:
Is it useful to define both the java versions of list operations (`get`, `contains`, `index_of`, etc) and the ruby ones? I've just been translating calls to them to the ruby versions